src/github_automation/core/project_item/base_project_item.py

```diff
@@ -57,33 +26,32 @@ def _extract_project_cards(project_cards):
     return card_id_project
 
 
-def parse_issue(github_issue):
-    return {
-        "id": github_issue['id'],
-        "title": github_issue['title'],
-        "number": github_issue['number'],
-        "assignees": _extract_assignees(github_issue.get('assignees', {}).get('edges', [])),
-        "pull_request": _get_pull_request(github_issue),
-        "labels": get_labels(github_issue.get('labels', {}).get('edges', [])),
-        "milestone": _get_milestone(github_issue),
-        "card_id_to_project": _extract_project_cards(github_issue.get('projectCards', {})),
-        "state": github_issue.get('state', '')
-    }
-
-
-class Issue(object):
-    def __init__(self, id: str, title: str, number: int, assignees: list = None, pull_request: PullRequest = None,
-                 labels: list = None, milestone: str = '', card_id_to_project: dict = None, priority_list: list = None,
-                 state: str = ''):
+def is_review_requested(pull_request_node):
+    if pull_request_node['reviewRequests']['totalCount'] or pull_request_node['reviews']['totalCount']:
+        return True
+
+    else:
+        return False
+
+
+def is_review_completed(pull_request_node):
+    return pull_request_node["reviewDecision"] == "APPROVED"
```

Comment:
what happens in case there are multiple reviewers where one approved and the other requested changes?

Reply:
`APPROVED` > `CHANGES_REQUESTED`  > `REVIEW_REQUIRED` https://docs.github.com/en/graphql/reference/enums#pullrequestreviewdecision
It's hard to tell by these docs, so I tested this. I'd guess that in that case, it'll still count as `APPROVED`.